src/algorithms/reco/JetReconstruction.cc

```diff
@@ -50,6 +49,28 @@ namespace eicrecon {
       m_log->error(" Unknown area type \"{}\" specified!", m_cfg.areaType);
       throw JException(out.what());
     }
+
+    // Choose jet definition based on no. of parameters
+    const int nParams = get_num_params();
```

Comment:
Not clear why we need to have two separate switches to exchange nParams value when you can just check for algorithm type right here.

Reply:
Ah! You're right! When I first coded it up I wanted to be _really_ explicit about what we were switching on, but just checking the algorithm type & some judicious comments will accomplish the same thing

Reply:
Looks much clearer now, thanks!